### PR TITLE
Refactor to enable manual lifecycle management

### PR DESCRIPTION
**Pull request by bdarfler**
We want to support running dynamodb in multiple test configurations (namely fun / it tests). This PR makes wiring a bit more manual but also more flexible.

### CODE REVIEW

File: src/main/scala/com/localytics/sbt/dynamodb/Utils.scala [hunk 1]
```diff
@@ -30,11 +24,23 @@ private[dynamodb] object Utils {
...
     }
   }
 
+  def killPidCommand(pid: String): String = {
+    val osName = System.getProperty("os.name") match {
+      case n: String if !n.isEmpty => n
+      case _ => System.getProperty("os")
+    }
+    if (osName.toLowerCase.contains("windows")) {
+      s"Taskkill /PID $pid /F"
```
Comment (jschlather) [inline src/main/scala/com/localytics/sbt/dynamodb/Utils.scala]: This brings back the memories. At my last job I wrote an Ant task that made a windows command to call taskkill on a Jrun executable.

Comment (bdarfler) [inline src/main/scala/com/localytics/sbt/dynamodb/Utils.scala]: thankfully someone else contributed this code, go OSS

**Comment by jsolmon** [comment]
This looks good to me so far. :+1:

**Comment by bdarfler** [comment]
Thanks @jsolmon! @joshcough let me know when you get a chance to look this over.

**Comment by joshcough** [comment]
@bdarfler I will definitely have a chance to do this tomorrow morning.

File: README.md [hunk 2]
```diff
@@ -15,18 +15,25 @@ Add the following to your `project/plugins.sbt` file:
...
-The following represents the minimum amount of code required in a `build.sbt` to use sbt-dynamodb.
 
-To use the dynamodb settings in your project, add `DynamoDBLocal.settings` to your build, set the directory to use for the DynamoDB Local jar and have your tests depend on starting the DynamoDB Local instance.
+To have DynamoDB Local automatically start and stop around your tests
 
 ```
-DynamoDBLocal.settings
+test in Test <<= (test in Test).dependsOn(startDynamoDBLocal)
+testOptions in Test += Tests.Cleanup(() => stopDynamoDBLocal.value)
+```
```
Comment (joshcough) [inline README.md]: Sweet.

**Comment by joshcough** [comment]
:shipit:

**Pull request merged**